client/src/js/components/bhFindPatient.js

```diff
@@ -65,16 +66,21 @@ function FindPatientComponent(Patients, AppCache, Notify, SessionService, bhCons
     vm.showSearchView = true;
     vm.loadStatus = null;
 
-
     // fetch the initial setting for the component from appcache
     loadDefaultOption(cache.optionKey);
 
     // call the onRegisterApi() callback with the
     vm.onRegisterApi({
-      api : { reset: vm.reset, searchByUuid: searchByUuid },
+      api : { reset : vm.reset, searchByUuid },
     });
   };
 
+  vm.$onChanges = function onChanges(changes) {
```

Comment:
This change should be accompanied by a test in the [bhFindPatient](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/test/client-unit/components/bhFindPatient/controller.spec.js) controller tests.